Sources/SwiftGenKit/Parsers/StringsParser.swift

```diff
@@ -37,23 +40,48 @@ public final class StringsParser: Parser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parse(path: Path) throws {
-    let name = path.lastComponentWithoutExtension
+    if path.isDirectory {
 
-    guard tables[name] == nil else {
-      throw StringsParserError.duplicateTable(name: name)
-    }
-    guard let data = try? path.read() else {
-      throw StringsParserError.failureOnLoading(path: path.string)
-    }
+      if try hasMultipleLprojSubdirectories(path: path) {
+        throw StringsParserError.multipleLprojDirectories
```

Comment:
I'm wondering if this really needs to be a terminating error? Maybe in some special situations in some projects that could be valid (e.g. `cn.lproj` having all the keys for the strings used in the Chinese version of the app but `en.lproj` having only some keys that were not in `cn.lproj` for features only available to English speaking countries? Idk… that seems unlikely for sure but I've seen so many strange setups 😅

So maybe instead just use `log(.warning, "the message here")` (or whatever I called that global method I use to print warnings in the console I'm not sure if the name I used and not handy to check from phone 😉)

Reply:
ok, I agree